log4j2-propagation/src/test/java/nl/talsmasoftware/context/log4j2/threadcontext/Log4j2ThreadContextManagerTest.java

```diff
@@ -121,26 +127,26 @@ void testGetActiveContext_close() {
         ThreadContext.push("stack2");
 
         // Should have no effect
-        Log4j2ThreadContextManager.INSTANCE.getActiveContext().close();
+        Log4j2ThreadContextManager.provider().getActiveContext().close();
 
         Map<String, String> expectedMap = new HashMap<String, String>();
         expectedMap.put("map1", "value1");
         expectedMap.put("map2", "value2");
-        assertEquals(expectedMap, ThreadContext.getContext());
+        assertThat(ThreadContext.getContext(), equalTo(expectedMap));
 
         List<String> expectedStack = Arrays.asList("stack1", "stack2");
-        assertEquals(expectedStack, ThreadContext.getImmutableStack().asList());
+        assertThat(ThreadContext.getImmutableStack().asList(), equalTo(expectedStack));
     }
 
     @Test
-    void test_ThreadContextPropagation() throws ExecutionException, InterruptedException {
+    void testThreadContextPropagation() throws ExecutionException, InterruptedException {
```

Comment:
Just for reference, the reason why I added an underscore is because this is not testing a specific method with name `threadContextPropagation`, but instead tests a use-case.
Though I do not mind renaming the methods.

Reply:
Thanks for the clarification. I named the use-case tests differently (a bit longer) to make their intent clearer.